src/makeCalculator.js

```diff
@@ -5,6 +5,47 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+
+    operate(operation, number = 0) {
+      operation.call(this, number);
+
+      return this;
+    },
+
+    add(number) {
+      this.result += number;
+
+      return this;
+    },
+
+    subtract(number) {
+      this.result -= number;
+
+      return this;
```

Comment:
The `subtract` method is correctly using the `-=` operator, following the checklist's code style recommendations.